webapp/javascript/pages/TagExplorerView.tsx

```diff
@@ -396,9 +413,9 @@ function Table({
       label: `${groupByTag === '' ? 'App' : 'Tag'} name`,
```

Comment:
Can we make this be "Application" instead of "App name" so that it's consistent with what we call it at the top:
![image](https://user-images.githubusercontent.com/23323466/199514486-52a2706c-ab05-4946-82dc-19aed6d502d5.png)